js/visualizations/eventTypeVis.js

```diff
@@ -890,100 +1002,66 @@ class SimulationDistance {
     }
 
     getTNWScore(simulation1, simulation2) {
-        console.log(simulation1, simulation2);
+        // console.log(simulation1, simulation2);
 
         // first sequence is vertical
         // second sequence is horizontal
         var matrix = []
-        const GAP_SCORE = -1
-        const MATCH_SCORE = 1
-        const MAX_OFFSET_PENALTY = -1
+        const GAP_PENALTY = -4
+        const MISMATCH_PENALTY = -1
+        const MATCH_REWARD = 1
+        const MAX_OFFSET_PENALTY = -3
 
         // initialize first row
         let firstRow = []
         for (let i=0; i<simulation2.length+1; i++) {
-            // firstRow[i] = i * GAP_SCORE
-            firstRow[i] = { cellMax: i*GAP_SCORE, arrowImage: null}
+            firstRow[i] = i * GAP_PENALTY
         }
         matrix.push(firstRow)
 
         // initialize first column
         for (let i=1; i<simulation1.length+1; i++){
             let row = Array(simulation2.length+1).fill(null, 1, simulation2.length+1)
-            // row[0] = i * GAP_SCORE
-            row[0] = { cellMax: i*GAP_SCORE, arrowImage: null }
+            row[0] = i * GAP_PENALTY
             matrix.push(row)
         }
 
         // fill in the rest of the table
         for (let i=1; i<simulation1.length+1; i++){
             for (let j=1; j<simulation2.length+1; j++){
-                let arrowImage = 'd.png'
-                let match = matrix[i-1][j-1].cellMax + MATCH_SCORE + this.offsetPenalty(simulation1[i], simulation2[j], MAX_OFFSET_PENALTY)
-                let vGap = matrix[i-1][j].cellMax + GAP_SCORE 
-                let hGap = matrix[i][j-1].cellMax + GAP_SCORE
-                let cellMax
-                if (match >= vGap && match >= hGap) {
-                    cellMax = match
-                    arrowImage = 'd.png'
-                    if (match === vGap && match === hGap) arrowImage = 'dsu.png'
-                    if (match === vGap) arrowImage = 'du.png'
-                    if (match === hGap) arrowImage = 'ds.png'
-                }
-                else if (vGap >= match && vGap >= hGap) {
-                    cellMax = vGap
-                    arrowImage = 'u.png'
-                    if (vGap === match && vGap === hGap) arrowImage = 'dsu.png'
-                    if (vGap === match) arrowImage = 'du.png'
-                    if (vGap === hGap) arrowImage = 'su.png'
+                
+                let cellMax = matrix[i-1][j-1] + MATCH_REWARD + this.offsetPenalty(simulation1[i], simulation2[j], MAX_OFFSET_PENALTY)
```

Comment:
this should be simulation1[i-1], simulation2[j-1] 
I found this bug probably after you forked this branch. sorry I should have told you what the problem was